src/run_benchmarks.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+success="TEST_SUCCESS"
+had_failures="0"
+tmp_file=".tmp_err_output"  # stderr of parser stored here
+
+reduce_path_to_test_name () {
+    local fullpath=$1
+    local filename="${fullpath##*/}" # strip the preceding path
+    test_name="${filename%.*}"       # strip the file type & set a global variable
+}
+
+run_bench() {
+    # stdout of parser
+    $(g++ -std=c++11 -pthread $file &> $tmp_file)
```

Comment:
sorry, that wasn't clear. it's shorthand for a `sed` command to replace `c++11` with `c++0x`. Because of the version of g++ that's on Travis, we need to use the `c++0x` flag.